Sming/Services/libb64/cdecode.c

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Please leave the coding style of the files under Sming/Services/libb64 intact. This way we can easier track changes with the upstream version of libb64 and update when needed.